chp02_forces/NOC_2_05_fluidresistance/sketch.js

```diff
@@ -29,7 +27,9 @@ function setup() {
 }
 
 function draw() {
-  background(127);
+  background(255);
+  fill(0);
+  text("click to reset",10,100);
```

Comment:
No worries, you'll have to add new commits that revert if you want the fame and glory that comes with having your pull request merged!